src/App.js

```diff
@@ -2,16 +2,19 @@ import React from 'react';
 import { HashRouter, Switch, Route } from "react-router-dom";
 import Main from './screens/Main/Main';
 import History from './screens/History/History';
+import { ScrollToTop } from '../src/components/ScrollToTop/ScrollToTop';
 import "./index.scss";
 
 function App() {
   return (
     <HashRouter>
       <Switch>
         <Route path='/History'>
+          <ScrollToTop />
```

Comment:
Well done! However, I recommend you wrapping the View components (Main, History) instead of adding yours before.

<ScrollToTop><Main/></ScrollToTop>

This will help identifying that it has been included here for that route.

Reply:
I think that it does'nt work.